libc/src/math/generic/ceilf.cpp

```diff
@@ -12,6 +12,12 @@
 
 namespace LIBC_NAMESPACE {
 
-LLVM_LIBC_FUNCTION(float, ceilf, (float x)) { return fputil::ceil(x); }
+LLVM_LIBC_FUNCTION(float, ceilf, (float x)) {
+#ifdef __LIBC_USE_BUILTIN_CEIL_FLOOR_TRUNC
```

Comment:
We shouldn't use double underscores for the name.

Reply:
libc/cmake/modules/LLVMLibCCompileOptionRules.cmake defines `__LIBC_RISCV_USE_FMA` and `__LIBC_EXPLICIT_SIMD_OPT`. I guessed that a macro defined through compile options should have `__`.